### PR TITLE
Blog Local Storage vs Session Storage vs Cookies added

### PR DESCRIPTION
**Pull request by FuzzySid**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [ x] You created a branch from `dev` branch.
- [x ] You PR is raised on `dev` branch and not on`master`.
- [x ] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x ] You have read the [Contribution Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x ] Checked for any duplicate blog entry.
- [x ] Checked for all grammatical errors in the blog.
- [x ] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x ] The content is not copy-pasted from anywhere else on the Internet.

### CODE REVIEW

File: content/guest-post/local-storage-vs-session-storage-vs-cookies/index.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+Consequently, you can retrieve an item using the `getItem()` function:
+
+```javascript
+localStorage.getItem("id")
+//'123'
+```
+
+### Usecase
+
+Local storage has a number of uses due to the fact that data inside it has no default expiry. Think about all those scenarios where you want to store some global data that's accessed oftenly in your application.
```
Comment (Akash-Patil) [inline content/guest-post/local-storage-vs-session-storage-vs-cookies/index.md:216]: I think `oftenly` should be `often` here.

File: content/guest-post/local-storage-vs-session-storage-vs-cookies/index.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+## Storing Your JWT/Auth Token
+
+Data such as JWT or Auth token should not be stored in browser storage because they can be accessed by any client side JavaScript running in the browser. This means that if your application somehow leaves an XSS vulnerability, your user's authentication token could be easily leaked to the attacker.
+
+The attacker could then make false requests, modify your user's data in the database, and do a lot of damage for your application as well as users. Hence, it's always best to store JWTs in http only cookies. Http only cookies are special cookies that cannot be accessed by client side JavaScript. This way they're secure against XSS attacks.
+
+Also, authentication token is often refreshed on expiry and one can use cookies TTL easily to manage that. For simpler cases, one can also store JWT inside regular cookies by setting a TTL.
+
+But all in all, authentication itself can be a tricky subject. If you're looking for a no-code identity platform that can seamlessly handle authentication for your application, consider using [login radius](https://www.loginradius.com/).
```
Comment (Akash-Patil) [inline content/guest-post/local-storage-vs-session-storage-vs-cookies/index.md:258]: Please change `login radius` to `LoginRadius`.

**Review by Akash-Patil — DISMISSED** [review]
Hi @FuzzySid , 

I have finished the technical review for the blog. The blog is well written and there are no changes required from a technical standpoint. 
However, a couple of minor text changes have to be made. Please have a look at the comments.

Regards,
@Akash-Patil

**Comment by FuzzySid** [comment]
Hi @Akash-Patil 
Thanks for letting me know. I'll need to create a separate PR for these changes right?

**Comment by raghunath-r-a** [comment]
Hi @FuzzySid As these are small changes, I'll change them directly and publish your blog tomorrow.

**Pull request merged**